### PR TITLE
minor Blazor Identity fixes

### PR DESCRIPTION
**Pull request by deepchoudhery**
cherry-picking #2997
- some more additional fixes, mostly missing logic as the branches were inconsistent.

### CODE REVIEW

File: scripts/install-aspnet-codegenerator.cmd [hunk 1]
```diff
@@ -1,5 +1,5 @@
-set VERSION=9.0.0-dev
-set DEFAULT_NUPKG_PATH=%userprofile%/.nuget/packages
+set VERSION=9.0.1-dev
+set DEFAULT_NUPKG_PATH=C:\Nuget
```
Comment (tlmii) [inline scripts/install-aspnet-codegenerator.cmd:2]: This is probably not what we want, right? (I note it's what was already in the branch so it wasn't a change there)

File: scripts/install-aspnet-codegenerator.cmd [hunk 2]
```diff
@@ -1,5 +1,5 @@
-set VERSION=9.0.0-dev
-set DEFAULT_NUPKG_PATH=%userprofile%/.nuget/packages
+set VERSION=9.0.1-dev
```
Comment (tlmii) [inline scripts/install-aspnet-codegenerator.cmd:1]: Should this be bumped to 9.0.1-dev now if we haven't shipped 9.0.0 yet? Its just for testing I guess so doesn't matter much, just curious the intent.

File: scripts/install-aspnet-codegenerator.cmd (hunk 1 above)
Comment (deepchoudhery) [reply scripts/install-aspnet-codegenerator.cmd:2]: on my devbox, the `USERPROFILE` path does not work, I'll just add it to search for both paths. Or check if there is a default nuget path environment variable. Will fix this in the next PR

File: scripts/install-aspnet-codegenerator.cmd (hunk 2 above)
Comment (deepchoudhery) [reply scripts/install-aspnet-codegenerator.cmd:1]: yep left over from testing sorry

File: scripts/install-aspnet-codegenerator.cmd (hunk 1 above)
Comment (tlmii) [reply scripts/install-aspnet-codegenerator.cmd:2]: If you mean an actual devbox and not just your dev box, devbox overrides the nuget package location by setting the NUGET_PACKAGES environment variable, which overrides the default `%userprofile%\.nuget\packages` location (for me it points to `q:\.tools\.nuget\packages\`).

**Review by tlmii — APPROVED** [review]

**Pull request merged**